proto/tendermint/types/types.proto

```diff
@@ -125,11 +125,14 @@ message EvidenceList {
   repeated Evidence evidence = 1 [(gogoproto.nullable) = false];
 }
 
-// Blob defines a chunk of data that is attributed to a namespace and under
-// usual circumstances, ends up published on-chain.
+// Blob (named after binary large object) is a chunk of data submitted by a user
+// to be published to the Celestia blockchain. The data of a Blob is published
+// to a namespace_id and is encoded into shares based on the format specified by
+// share_version.
```

Comment:
we need more comments like these 👏 👏

Reply:
agree! @rootulp is particularly gifted at technical communication